### DISCUSSION TITLE
Actions on Pull Request against another Pull Request?

### DISCUSSION THREAD

**Discussion by mrcasals** [community/community#25142] [Actions]
<p>Hi!</p>
<p>We’re using GitHub Actions to run our test suite. Here’s our config:</p>
<pre><code>on:
  push:
    branches:
      - develop
      - master
      - release/*
      - "*-stable"
  pull_request:
    branches:
      - "*"
</code></pre>
<p>This has been working fine for us until now. We have to add a big feature, so wer thought about splitting it into different PRs. We’d create a base PR for the whole feature, then smaller PRs against that base PR.</p>
<p>Base PR: <a href="https://github.com/decidim/decidim/pull/5925" rel="nofollow noopener">https://github.com/decidim/decidim/pull/5925</a></p>
<p>Smaller PR: <a href="https://github.com/decidim/decidim/pull/5924" rel="nofollow noopener">https://github.com/decidim/decidim/pull/5924</a></p>
<p>(note the base branches on each PR; base branch was created later)</p>
<p>The problem is that we don’t get any check for the smaller PR, it doesn’t seem to be firing.</p>
<p>Any idea what’s going on and how to fix the config so the tests run?</p>
<p>Thanks!</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/mrcasals">@mrcasals</a> ,</p>
<p>I noticed that you are using the below configuration in your workflow:</p>
<pre><code>on:
  pull_request:
    branches:
      - "*"
</code></pre>
<p>The branches filter pattern <strong>‘*’</strong> only matches the branch names that don’t contain a slash (/). This is the root cause of the problem you are facing.</p>
<p>If you want to match all branch names, include the names that contain slashes (/), you can use one of the below two configurations:</p>
<pre><code>on:
  pull_request:
    branches:
      - "**"
</code></pre>
<p>because  <strong>‘**’</strong> is default branches filter pattern when you don’t set any branches filter, so you also can directly use,</p>
<pre><code>on:
  pull_request:
</code></pre>
<p>More details, you can reference here: <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags" rel="nofollow noopener">https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags</a></p>

**Comment by mrcasals** [comment]
<p>Ah, I missed that section, thank you very much!</p>
